lib/components/viewers/stop-viewer.js

```diff
@@ -303,11 +307,13 @@ class StopViewer extends Component {
             />
           )}
         </div>
-        <FromToLocationPicker
-          label
-          onFromClick={this._onClickPlanFrom}
-          onToClick={this._onClickPlanTo}
-        />
+        <span role="group">
```

Comment:
Thoughts on incorporating this group role into the `FromToLocationPicker` itself?

Reply:
I was talking about this with Adriana too -- we should probably do it if both of you came up with the same idea independently.

Reply:
Amy and I have the same mind haha. But yeah up to you